keda/templates/22-metrics-deployment.yaml

```diff
@@ -42,6 +42,7 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ .Values.serviceAccount.name }}
+      automountServiceAccountToken: true
```

Comment:
Can you move this to a value with default property `true` instead please?

Reply:
Yes I can do that. I am not sure if this makes sense as you need that to set to true for having API Access. If you would set it to false KEDA would not work anymore. Please confirm if I should change it.